package/dart-sass.sh

```diff
@@ -8,12 +8,15 @@
 # package`.
 
 function follow_links() {
-  file="$1"
+  local file="$1"
+
+  cd $(dirname "$file")
   while [ -h "$file" ]; do
     # On Mac OS, readlink -f doesn't work.
     file="$(readlink "$file")"
+    cd $(dirname "$file")
   done
-  echo "$file"
+  echo "$(pwd -P)/$(basename "$file")"
```

Comment:
For the 1st question, sorry for the confusion, this script isn't deployed for ```dart``` package with Homebrew. Obviously, it's mandatory for ```dart-sass``` package.
```
$ ls /usr/local/bin/dart
/usr/local/bin/dart -> ../Cellar/dart/1.21.1/bin/dart
$ file /usr/local/Cellar/dart/1.21.1/bin/dart
/usr/local/Cellar/dart/1.21.1/bin/dart: Mach-O 64-bit executable x86_64
```

> Why doesn't repeatedly readlinking work? 
> Why does cding fix it?

I don't really know why ```readlink``` tool doesn't do the job.
Changing directory is working because Homebrew symlinks are relative (not absolute).

> Doesn't it change the user's working directory in a way they won't want?

The script forks shell environment when launching (because of the "shebang").
Only exported values (with ```export``` command) will be "visible" at the end of the script.
In other words, the "cd" commands in the script won't affect the user's environment: the current working directory is unchanged.

> Why is local file required?

It's a best practice to put ```local``` to reduce the scope of the variable.
Otherwise, the variable is global by default on shell script.
https://google.github.io/styleguide/shell.xml#Use_Local_Variables